helm_deploy/licences/charts/gotenberg/values.yaml

```diff
@@ -0,0 +1,9 @@
+# This is a YAML-formatted file.
```

Comment:
The values entered here are defaults.  They're all overridden in values-dev.yaml, values-preprod.yaml and values-prod.yaml